lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -2072,9 +2078,19 @@ instance ToJSON (ApiT BoundType) where
 instance FromJSON (ApiT BoundType) where
     parseJSON = fmap ApiT . genericParseJSON defaultSumTypeOptions
 
-instance DecodeAddress t => FromJSON (PostTransactionData t) where
+instance FromJSON ApiSerialisedTransaction where
+    parseJSON = fmap ApiSerialisedTransaction . fromBaseText Base16
```

Comment:
I don't think it matters that much. I thought hex might be slightly easier to use. Base64 is 1.33× raw size and hex is 2× raw size - both acceptable for transactions given their max size. Anyway I have changed it to Base64 because the sign metadata endpoint uses Base64, as you say. I also changed the encoding accepted by the `cardano-wallet transaction submit` CLI from hex to base64 to align with this.